tests/legacy-cli/e2e/utils/registry.ts

```diff
@@ -1,17 +1,24 @@
-import { ChildProcess, spawn } from 'child_process';
-import { copyFileSync, mkdtempSync, realpathSync } from 'fs';
+import { spawn } from 'child_process';
+import { mkdtempSync, realpathSync } from 'fs';
 import { tmpdir } from 'os';
 import { join } from 'path';
-import { writeFile } from './fs';
+import { getGlobalVariable } from './env';
+import { writeFile, readFile } from './fs';
 
-export function createNpmRegistry(withAuthentication = false): ChildProcess {
+export async function createNpmRegistry(
+  port: number,
+  httpsPort: number,
+  withAuthentication = false,
+) {
   // Setup local package registry
   const registryPath = mkdtempSync(join(realpathSync(tmpdir()), 'angular-cli-e2e-registry-'));
 
-  copyFileSync(
+  let configContent = await readFile(
     join(__dirname, '../../', withAuthentication ? 'verdaccio_auth.yaml' : 'verdaccio.yaml'),
-    join(registryPath, 'verdaccio.yaml'),
   );
+  configContent = configContent.replace(/\$\{HTTP_PORT\}/g, String(port));
+  configContent = configContent.replace(/\$\{HTTPS_PORT\}/g, String(httpsPort));
```

Comment:
NIT 
```suggestion
 configContent = configContent
  .replace(/\$\{HTTP_PORT\}/g, String(port))
  .replace(/\$\{HTTPS_PORT\}/g, String(httpsPort));
```

Reply:
👍 but looks like it's too late...

Reply:
@jbedard, feel free to include this suggested change in on of your upcoming PRs.